check_python.sh

```diff
@@ -65,7 +65,7 @@ if [ -n "${project_check_module}" ]; then
       echo "Critical error running ${project_check_module}"
       echo command output was
       cat check_module.out
-      exit ${rc}
+      exit 1
```

Comment:
The lint fix was just to quote `${rc}` wasn't it?

Reply:
The lint fix was to quote $rc, I thought adding the {} would fix it but obviously not.  We're not actually trying to convey any information thought the code, just a pass/fail so this is just as reliable.